### PR TITLE
Replace and/or remove refrences to ludeeus

### PR DESCRIPTION
**Pull request by ludeeus**
# Proposed Changes

> (Describe the changes and rationale behind them)

It's now been almost 2.5 years since my last commit to this repository, which means that I'm clearly not maintaining it, and the credit for that should not be given to me.

## Related Issues

> ([Github link][autolink-references] to related issues or pull requests)

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +76,7 @@ our [GitHub Repository][repository].
 
 MIT License
 
-Copyright (c) 2018-2021 Joakim Sørensen
+Copyright (c) 2018-2021 Home Assistant Community Add-ons project
```
Comment (ludeeus) [inline README.md:79]:
```suggestion
Copyright (c) 2018-2019 Joakim Sørensen
Copyright (c) 2019-2022 Franck Nijhof
```

File: tautulli/Dockerfile [hunk 2]
```diff
@@ -70,11 +70,11 @@ LABEL \
     io.hass.arch="${BUILD_ARCH}" \
     io.hass.type="addon" \
     io.hass.version=${BUILD_VERSION} \
-    maintainer="Joakim Sørensen @ludeeus <ludeeus@gmail.com>" \
+    maintainer="Home Assistant Community Add-ons project" \
```
Comment (ludeeus) [inline tautulli/Dockerfile:73]:
```suggestion
    maintainer="Franck Nijhof <frenck@addons.community>" \
```

File: tautulli/Dockerfile [hunk 3]
```diff
@@ -70,11 +70,11 @@ LABEL \
     io.hass.arch="${BUILD_ARCH}" \
     io.hass.type="addon" \
     io.hass.version=${BUILD_VERSION} \
-    maintainer="Joakim Sørensen @ludeeus <ludeeus@gmail.com>" \
+    maintainer="Franck Nijhof <frenck@addons.community>" \
     org.opencontainers.image.title="${BUILD_NAME}" \
     org.opencontainers.image.description="${BUILD_DESCRIPTION}" \
     org.opencontainers.image.vendor="Home Assistant Community Add-ons" \
-    org.opencontainers.image.authors="Joakim Sørensen @ludeeus <ludeeus@gmail.com>" \
+    org.opencontainers.image.authors=""Franck Nijhof <frenck@addons.community>" \
```
Comment (frenck) [inline tautulli/Dockerfile:77]:
```suggestion
    org.opencontainers.image.authors="Franck Nijhof <frenck@addons.community>" \
```

**Review by frenck — APPROVED** [review]
Thanks, @ludeeus 👍

**Pull request merged**